src/public/index.html

```diff
@@ -20,10 +20,10 @@
         <nav>
           <ul class="no-bullets li">
             <li>
-              <a href="/about/">About</a>
+              <a href="/about.html">About</a>
             </li>
             <li>
-              <a href="/contact/">Contact</a>
+              <a href="/contact.html">Contact</a>
```

Comment:
Nice! Don't forget to also fix this in `about.html`. (This is why we're going to use templates and dynamically render our HTML soon...)